regression/benchmark_scripts/utils.py

```
@@ -0,0 +1,188 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: LicenseRef-NvidiaProprietary
+#
+# NVIDIA CORPORATION, its affiliates and licensors retain all intellectual
+# property and proprietary rights in and to this material, related
+# documentation and any modifications thereto. Any use, reproduction,
+# disclosure or distribution of this material and related documentation
+# without an express license agreement from NVIDIA CORPORATION or
+# its affiliates is strictly prohibited.
+
+from cuopt_server.utils.utils import build_routing_datamodel_from_json
+from cuopt import routing
```

Flake8 (7.3.0)

[error] 'cuopt.routing' imported but unused

(F401)